sr_multi_moveit/sr_multi_moveit_config/launch/moveit.rviz

```diff
@@ -1,23 +1,26 @@
 Panels:
```

Comment:
Can we revert this to kinetic-devel moveit.rviz?

Reply:
Fixed in https://github.com/shadow-robot/sr_interface/pull/419/commits/69bc13be8e64f769c2179bda3d2bfc388b4bc071